Doppler.ContactPolicies.Business.Logic/UserApiClient/Services/UserFeaturesService.cs

```diff
@@ -0,0 +1,43 @@
+using Doppler.ContactPolicies.Business.Logic.DTO;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Business.Logic.UserApiClient.Services
+{
+    public class UserFeaturesService : IUserFeaturesService
+    {
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IConfiguration _configuration;
+        private readonly ILogger<UserFeaturesService> _logger;
+
+        public UserFeaturesService(IHttpClientFactory clientFactory, IConfiguration configuration, ILogger<UserFeaturesService> logger)
+        {
+            _clientFactory = clientFactory;
+            _configuration = configuration;
+            _logger = logger;
+        }
+        public async Task<bool> GetUserContactPoliciesFeatureAsync(string accountName)
+        {
+            try
+            {
+                var baseUri = _configuration.GetSection("UsersApiURL").Value;
+                var uri = new Uri(baseUri + $"/accounts/{accountName}/features");
+
+                var client = _clientFactory.CreateClient("users-api");
+                var responseString = await client.GetStringAsync(uri);
+
+                var features = JsonConvert.DeserializeObject<Features>(responseString);
+                return features.ContactPolicies;
+            }
+            catch (HttpRequestException httpRequestException)
```

Comment:
Por qué no capturamos la excepción general? Vamos a tomar alguna acción en particular para la excepción `HttpRequestException`? 
```suggestion
            catch (Exception exception)
```